hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/HoodieSparkUtils.scala

```diff
@@ -131,6 +132,16 @@ object HoodieSparkUtils extends SparkAdapterSupport with SparkVersionsSupport wi
   def injectSQLConf[T: ClassTag](rdd: RDD[T], conf: SQLConf): RDD[T] =
     new SQLConfInjectingRDD(rdd, conf)
 
+  def maybeWrapDataFrameWithException(df: DataFrame, exceptionClass: String, msg: String, shouldWrap: Boolean): DataFrame = {
+    if (shouldWrap) {
+      HoodieUnsafeUtils.createDataFrameFromRDD(df.sparkSession, injectSQLConf(df.queryExecution.toRdd.mapPartitions {
```

Comment:
https://github.com/apache/hudi/pull/6352 alexey says that "Yes, it will propagate to all RDDs in the execution chain (up to a shuffling point)". So I guess the question is if there is a shuffling point?

Reply:
My doubt is if wrapping the df with exception needs the SQL conf injection.  So you're saying this needs SQL conf injection so the configs can still be propagated to the executors.

Reply:
Let's keep an eye on the DAG and Spark execution when this is rolled out.  I think this is OK for now.